core/server/api/v2/utils/validators/input/schemas/posts-add.json

```diff
@@ -0,0 +1,20 @@
+
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "posts.add",
+    "title": "posts.add",
+    "description": "Schema for posts.add",
+    "type": "object",
+    "additionalProperties": false,
+    "properties": {
+        "posts": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "allOf": [{"$ref": "posts#/definitions/post"}],
+                "required": ["title", "authors"]
```

Comment:
@kirrg001 a question about "authors" here. Should we maybe assign some default author before going into validation stage? This would make it simpler to have the same logic for integration request and regular authenticated user request.

Reply:
I don't understand what you mean with "default" author?

Reply:
What I meant was having an 'authors' field being filled out in `data` when the request is made by authenticated user and nothing is present in that field.